fhir-server/src/test/java/com/ibm/fhir/server/util/FHIRRestHelperTest.java

```diff
@@ -141,10 +123,10 @@ public void testTransactionBundlePostSingle() throws Exception {
         assertNotNull(responseBundle);
         assertEquals(1, responseBundle.getEntry().size());
         Bundle.Entry entry = responseBundle.getEntry().get(0);
-        assertEquals(ALL_OK, entry.getResource());
```

Comment:
@michaelwschroeder the order of the assert arguments in this class are all messed up.  I fixed the ones that I touched, but I *did not* wholistically fix them in the file (nor check for it anywhere else in the project).
I think its because we use JUnit style asserts in some places and testng ones in others.  Something to watch for going forward.